GVFS/GVFS.UnitTests/Maintenance/LooseObjectStepTests.cs

```diff
@@ -143,9 +143,10 @@ public void LooseObjectsCount()
             this.TestSetup(DateTime.UtcNow.AddDays(-7));
 
             LooseObjectsStep step = new LooseObjectsStep(this.context, requireCacheLock: false, forceRun: false);
-            int count = step.CountLooseObjects();
+            step.CountLooseObjects(out int count, out long size);
 
             count.ShouldEqual(3);
+            size.ShouldEqual(11);

```

Comment:
I think a small comment on how we got to 3 / 11, would be helpful for readability